src/main/java/hudson/plugins/sloccount/SloccountBuildAction.java

```diff
@@ -37,6 +45,12 @@ public String getDisplayName() {
     public String getUrlName() {
         return URL_NAME;
     }
+    
+    @Override  
+    public Collection<? extends Action> getProjectActions() {  
+        return this.projectActions;  
```

Comment:
Honestly I do not know how many time and whet getProjectActions can be called.
So to not create list and ProjectActions to many time I decided to create and add it ones on onLoad/onAttached and just return list on getProjectActions.